<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>319</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>446</width>
    <height>319</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>446</width>
    <height>319</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_img">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>43</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>260</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>Вперёд</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_task">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Во сколько лет вы научились читать?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_selected">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>240</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрано:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>192</height>
     </rect>
    </property>
    <property name="currentRow">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>младше 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 года</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>старше 6</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
